app/models/schools/configuration.rb

```diff
@@ -0,0 +1,80 @@
+# == Schema Information
+#
+# Table name: configurations
+#
+#  created_at       :datetime         not null
+#  dashboard_charts :json             not null
+#  id               :bigint(8)        not null, primary key
+#  school_id        :bigint(8)        not null
+#  updated_at       :datetime         not null
+#
+# Indexes
+#
+#  index_configurations_on_school_id  (school_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (school_id => schools.id) ON DELETE => cascade
+#
+
+module Schools
+  class Configuration < ApplicationRecord
+    belongs_to :school
+
+    delegate :meters, :meters_with_validated_readings, to: :school
+
+    def fuel_types_for_analysis
```

Comment:
Move these in to a separate new service which returns a simple object, we can then use that for asking things like solar_pv?